server/views/pages/component.njk

```diff
@@ -130,31 +130,6 @@
             No Team slack &#129301; - please provide one.
```

Comment:
This is just going to make the column really wide.

Reply:
This is not updated in this PR , so will leave it for now